### ISSUE TITLE
Strava correlations

### ISSUE THREAD

**Issue by commuterjoy** [commuterjoy/ltn#6]
https://www.strava.com/activities/4158354903

**Comment by milh0use** [comment]
https://strava.app.link/kw7fSm9moab

**Comment by milh0use** [comment]
https://strava.app.link/o8Pm3Ubnoab

**Comment by milh0use** [comment]
Check out my activity on Strava: https://strava.app.link/vJ1sMienoab

**Comment by commuterjoy** [comment]
@milh0use thanks, did you track a car when doing these?

**Comment by milh0use** [comment]
> @milh0use thanks, did you track a car when doing these? 

Yep, each time spotted a couple of number plates that started the route with me and kept an eye on them. The stop time in each case was when one of the cars reached the destination. The times en route won't be completely accurate as I needed to get a head start on places to keep a similar pace!

**Comment by commuterjoy** [comment]
ok, the Google data is looking on the pessimistic side still - eg. https://www.strava.com/activities/4163436134 - you clocked 8 mins, Google says 12 mins in traffic.

**Comment by milh0use** [comment]
Ah ha! Not true! You need to look at elapsed time, not moving time! I spent a lot of time not moving because of the traffic chaos. The elapsed time was 11:02 so only 58 seconds out. ~8%

**Comment by commuterjoy** [comment]
perfect!
